packages/medusa-source-shopify/src/services/shopify-product.js

```diff
@@ -106,15 +111,15 @@ class ShopifyProductService extends BaseService {
         }
       }
 
-      await this.redis_.addIgnore(data.id, "product.created")
+      await this.cacheService_.addIgnore(data.id, "product.created")
```

Comment:
Good catch, I think exposing clients would kinda break the interface because some clients don't have that functionality and some modules don't have clients at all. But in this case, I would suggest we reimplement this with a simple Map on service and keep the cache interface clean so it could be implemented across different technologies, wdyt?

Reply:
`this.cacheService` is an instance of `ShopifyCacheService` which is using a cache module as an underlying store but have this methods